aptos-move/block-executor/src/executor.rs

```diff
@@ -623,6 +626,10 @@ where
     ) -> Result<Vec<E::Output>, E::Error> {
         let num_txns = signature_verified_block.len();
         let init_timer = VM_INIT_SECONDS.start_timer();
+
+        // Shared throughput different sessions / txn executions for generating IDs via
+        // the view provided to the executor.
+        let shared_view_counter = RefCell::<u32>::new(0);
```

Comment:
Do we need this change here?

Reply:
I just realized I had added the placeholder for the counter, but the counter wasn't really shared :D 
Can do it in an earlier PR? or anywhere else?

Reply:
It's is fine :)